jxl/src/frame/quant_weights.rs

```diff
@@ -136,7 +134,7 @@ impl QuantEncoding {
                 if predefined as usize >= 1 {
```

Comment:
Remove this logic too? The error can never be triggered anyway.

Reply:
Reading 0 bits from the bitstream does nothing, please remove that.

Reply:
Oh, it's actually really 0 bits?

I thought it read something.

Removing it.